### PR TITLE
Multi geo1

### PR DESCRIPTION
**Pull request by SumitParikh**
Changed documentation URL in beta, Copied Beta properties to v1.0 and changed ChangeLog file

### CODE REVIEW

File: api-reference/beta/resources/group.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ This resource supports:
...
 |onPremisesSecurityIdentifier|String|Contains the on-premises security identifier (SID) for the group that was synchronized from on-premises to the cloud. Read-only. |
 |onPremisesSyncEnabled|Boolean|**true** if this object is synced from an on-premises directory; **false** if this object was originally synced from an on-premises directory but is no longer synced; **null** if this object has never been synced from an on-premises directory (default). Read-only. Supports $filter.|
-|preferredDataLocation|String|The preferred data location for the user. For more information see: [OneDrive for Business Multi-Geo tenant configuration](https://docs.microsoft.com/office365/enterprise/multi-geo-tenant-configuration)  and [Multi-Geo Capabilities in Exchange Online](https://docs.microsoft.com/office365/enterprise/multi-geo-capabilities-in-exchange-online).|
+|preferredDataLocation|String|The preferred data location for the group. For more information see:  [OneDrive Online Multi-Geo](https://docs.microsoft.com/sharepoint/dev/solution-guidance/multigeo-introduction).|
```
Comment (dkershaw10) [inline api-reference/beta/resources/group.md]: Minor nit: Probably fine, but it reads a bit funny to me.  It feels like the topic name should have an "introduction" or a "guidance" moniker as part of the link name.

File: api-reference/beta/resources/organization.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Only the read and update operations are supported on tenants; create and delete
...
 |displayName|String|The display name for the tenant.|
 |id|String|The unique identifier for the tenant. Inherited from [directoryObject](directoryobject.md). Key. Not nullable. Read-only.|
-|isMultipleDataLocationsForServicesEnabled|Boolean|**true** if organization is Multi-Geo enabled; **false** if organization is not Multi-Geo enabled; **null** (default). Read-only. For more information see: [Plan for OneDrive for Business Multi-Geo](https://docs.microsoft.com/office365/enterprise/plan-for-multi-geo)  and [Multi-Geo Capabilities in Exchange Online](https://docs.microsoft.com/office365/enterprise/multi-geo-capabilities-in-exchange-online).|
+|isMultipleDataLocationsForServicesEnabled|Boolean|**true** if organization is Multi-Geo enabled; **false** if organization is not Multi-Geo enabled; **null** (default). Read-only. For more information see: [OneDrive Online Multi-Geo](https://docs.microsoft.com/sharepoint/dev/solution-guidance/multigeo-introduction).|
```
Comment (dkershaw10) [inline api-reference/beta/resources/organization.md]: Don't think Multi-Geo should be captilized.  Please fix.

**Comment by SumitParikh** [comment]
Hi Dan,

In Office documentation<https://docs.microsoft.com/sharepoint/dev/solution-guidance/multigeo-introduction> they capitalize Multi-Geo everywhere and hence I did the same in our documentation.





Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

**Comment by SumitParikh** [comment]
I can change the topic name to something like this. Does this sound good?

For more information see:  OneDrive Online Multi-Geo introduction<https://docs.microsoft.com/sharepoint/dev/solution-guidance/multigeo-introduction>.





Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

File: api-reference/beta/resources/group.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/resources/group.md]: Can you please edit the link text as follows: "For more information, see [OneDrive Online Multi-Geo]". Thanks.

**Comment by SumitParikh** [comment]
+ Dan

Hi Laura, is the suggestion to only remove “:” from the sentence?




Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

**Comment by SumitParikh** [comment]
Thoughts Dan?





Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

From: Sumit Parikh
Sent: Monday, August 13, 2018 9:44 AM
To: 'microsoftgraph/microsoft-graph-docs' <reply@reply.github.com>; microsoftgraph/microsoft-graph-docs <microsoft-graph-docs@noreply.github.com>; Dan Kershaw <dkershaw@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [microsoftgraph/microsoft-graph-docs] Multi geo1 (#2961)

I can change the topic name to something like this. Does this sound good?

For more information see:  OneDrive Online Multi-Geo introduction<https://docs.microsoft.com/sharepoint/dev/solution-guidance/multigeo-introduction>.





Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

**Comment by SumitParikh** [comment]
Hi Laura,

Can you confirm?



Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

From: Sumit Parikh
Sent: Monday, August 13, 2018 10:13 AM
To: microsoftgraph/microsoft-graph-docs <reply@reply.github.com>; microsoftgraph/microsoft-graph-docs <microsoft-graph-docs@noreply.github.com>; Dan Kershaw <dkershaw@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [microsoftgraph/microsoft-graph-docs] Multi geo1 (#2961)

+ Dan

Hi Laura, is the suggestion to only remove “:” from the sentence?




Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

File: api-reference/beta/resources/group.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/resources/group.md]: My reply to the email thread did not go through. I made the edit.

File: api-reference/beta/resources/organization.md (hunk 2 above)
Comment (Lauragra) [reply api-reference/beta/resources/organization.md]: Made edit here too.

**Comment by Lauragra** [comment]
FYI, the link text should match the topic title. I don’t recommend changing the link text unless you also plan to update the topic title….

It does appear that Multi-Geo is capitalized in the docs; is it an official product or sku name? In some instances where it’s capped, I think it should be lower case – i.e. “Multi-Geo tenant” can be “multi-geo tenant”.

Thanks,
Laura

**Comment by SumitParikh** [comment]
Thanks Laura I see your edits and they make sense. Multi-Geo is official feature name and hence I believe it is capped in the product documentation. Office owns this feature.



Thanks and Regards,
Sumit Parikh, Senior Program Manager
Microsoft Corporation
M: +1 425.241.5440

**Comment by lleonard-msft** [comment]
@Lauragra - This had merge conflicts and a validation error.  I've corrected both.  It looks ready to merge.  Your thoughts?

**Comment by Lauragra** [comment]
@lleonard-msft - Looks good to me! Feel free to merge.

**Comment by lleonard-msft** [comment]
Merged; thank you!

**Pull request merged**